freqtrade/optimize/hyperopt/hyperopt.py

```diff
@@ -37,7 +37,7 @@
 
 # Keep no more than SKOPT_MODEL_QUEUE_SIZE models
 # in the skopt model queue, to optimize memory consumption
-SKOPT_MODEL_QUEUE_SIZE = 10
+# SKOPT_MODEL_QUEUE_SIZE = 10
```

Comment:
if it's no longer needed - we can clean this up? (including the comment above i assume)

Reply:
yes